src/app/dataset/workflow/workflow.component.ts

```diff
@@ -120,21 +112,28 @@ export class WorkflowComponent implements OnInit {
     this.getWorkflow();
     this.formInitialised.emit(this.workflowForm);
 
-    this.newNotification = successNotification(this.translate.instant('workflowsavenew'), {
-      sticky: true
-    });
-    this.saveNotification = successNotification(this.translate.instant('workflowsave'), {
-      sticky: true
-    });
-    this.runningNotification = successNotification(this.translate.instant('workflowrunning'), {
-      sticky: true
-    });
-    this.invalidNotification = errorNotification(this.translate.instant('formerror'), {
-      sticky: true
-    });
-    this.gapInSequenceNotification = successNotification(this.translate.instant('gaperror'), {
-      sticky: true
-    });
+    const notificationConf = { sticky: true };
+
+    this.newNotification = successNotification(
+      this.translate.instant('workflowsavenew'),
+      notificationConf
+    );
+    this.saveNotification = successNotification(
+      this.translate.instant('workflowsave'),
+      notificationConf
+    );
+    this.runningNotification = successNotification(
+      this.translate.instant('workflowrunning'),
+      notificationConf
+    );
+    this.invalidNotification = errorNotification(
+      this.translate.instant('formerror'),
+      notificationConf
+    );
+    this.gapInSequenceNotification = successNotification(
+      this.translate.instant('gaperror'),
+      notificationConf
+    );
```

Comment:
Here is some repetition of code. Is there a way to reduce the code? Can this be refactored?

Reply:
We tried... we failed